server/bin/pbench-sync-satellite.sh

```diff
@@ -266,6 +266,13 @@ if [ -s ${state_change_log} ] ;then
     fi
 fi
 
+if [ $nerrs == 0 ] ; then
```

Comment:
Can you use the double-bracket form here?  Something like, `if [[ ${nerrs} == 0 ]]; then`?

Reply:
Sure :)